### DISCUSSION TITLE
Emote priority: Choosing between 7tv, bttv and ffz

### DISCUSSION THREAD

**Discussion by pr0gresiv** [SevenTV/chatterino7#43] [Ideas]
Hey,

Since 7tv implemented new halloween emotes it would be great to have a feature where u can choose between those emote applications. The problem is, that if a BTTV and (global) 7TV emote have the same name, the BTTV emote is always prioritised. Therefore it is not possible to use the new halloween emotes. It would be neat to have a feature, where you could change the priorization of these emotes.

I would really appreciate it if theres an opportunity to add this feature in Chatterino

**Comment by Nerixyz** [comment]
This is a feature request for [upstream Chatterino](https://github.com/Chatterino/chatterino2). Please open a discussion there (if there's no existing one).
